ui/apps/platform/src/Containers/ConfigManagement/Dashboard/DashboardPage.jsx

```diff
@@ -13,7 +13,7 @@ import ComplianceByControls from './widgets/ComplianceByControls';
 import UsersWithMostClusterAdminRoles from './widgets/UsersWithMostClusterAdminRoles';
 import SecretsMostUsedAcrossDeployments from './widgets/SecretsMostUsedAcrossDeployments';
 
-const ConfigManagementDashboardPage = () => {
+const DashboardPage = () => {
```

Comment:
I know that having a prefix is usually helpful when searching for components in the IDE. What are your thoughts on the effect on that workflow?

Reply:
Yeah, let’s ask the team.

Reply:
Updated.